src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private Random rand;
+    private ColorSupplier colorSupplier;
+
+    public Ball getRandomBall() {
+        return new Ball(rand.nextInt(100), colorSupplier.getRandomColor());

```

Comment:
100 must be a constant. Also create variables for rand.nextInt(100) and colorSupplier.getRandomColor(). This makes the code more readable